Microsoft.Recognizers.Text.DateTime/English/Utilities/EnlighDatetimeUtilityConfiguration.cs

```diff
@@ -21,28 +21,21 @@ public class EnlighDatetimeUtilityConfiguration : IDateTimeUtilityConfiguration
             "in",
         };
 
-        public static readonly List<string> AmStringList = new List<string>
-        {
-            "am",
-            "a.m.",
-            "a",
-        };
+        public static readonly string AmPrefix = "a";
+        public static readonly string PmPrefix = "p";
+        public static readonly string AmPmPrefix = "ampm";
 
-        public static readonly List<string> PmStringList = new List<string>

```

Comment:
I see. Please add that as a comment in the code in the method that uses startswith and the new token, so new people to the code quickly get the meaning.
Thanks.